modules/src/joke.py

```diff
@@ -1,14 +1,30 @@
 import requests
 from templates.text import TextTemplate
+from random import choice
+import config
 
 def process(input, entities=None):
     output = {}
     try:
+        '''
         r = requests.get('http://tambal.azurewebsites.net/joke/random')
         data = r.json()
         output['input'] = input
         output['output'] = TextTemplate(data['joke']).get_message()
-        output['success'] = True
+        output['success'] = True        
+        '''
+        
+        with open(config.JOKES_SOURCE_FILE,"r") as file:
+            dump = file.read() # read the whole file
+            jokes_list = dump.split("\'''")[2].split("\n")
```

Comment:
I can assure you this may seem hacky but its actually not. It's become like this because of the comment block in the jokes.txt file. I've essentially put a comment block for newcomers.